post_installation_configuration/cluster-tasks.adoc

```diff
@@ -96,3 +96,25 @@ Understand and configure Pod disruption budgets.
 
 include::modules/nodes-pods-pod-disruption-about.adoc[leveloffset=+2]
 include::modules/nodes-pods-pod-disruption-configuring.adoc[leveloffset=+2]
+
+[id="post-install-rotate-cloud-creds"]
+== Rotating cloud provider credentials
+Some organizations require the rotation of the cloud provider credentials that were used to install {product-title} after installation. The xref:../operators/operator-reference.adoc#cloud-credential-operator_red-hat-operators[Cloud Credential Operator (CCO)] manages cloud provider credentials and can be configured to operate in several different modes. The process for rotating cloud credentials depends on the mode that the CCO is configured to use.
+
+[id="cred-rotation-cco-mode-mint"]
+=== Rotating credentials with the Cloud Credential Operator in mint mode
+When the CCO is configured to operate in _mint mode_, it uses the provided admin-level cloud credential to create new credentials for each component with only the specific permissions that are required.
+
+To rotate credentials in mint mode, you must manually remove the component credentials that were created by the removed credential.
+
+. Update the admin credentials root secret for your cloud provider.
+
+. Delete each component secret that is referenced by the individual `CredentialsRequest` objects.
+
+The CCO uses the new admin-level cloud credential to create new credentials for each component.
+
+[id="cred-rotation-cco-mode-passthrough"]
+=== Rotating credentials with the Cloud Credential Operator in passthrough mode
+When the CCO is configured to operate in _passthrough mode_, it passes the provided cloud credential to the components that request cloud credentials. The credential must have permissions to perform the installation and complete the operations that are required by components in the cluster, but does not need to be able to create new credentials.
+
+After rotating the passthrough mode cloud credential, it can take up to 11 hours to propagate the new credential to all of the components that need it. During this time, the cluster might not be fully operational. To reduce the time to propagate the new credential, delete the CCO pod. Deleting the CCO pod causes the CCO to immediately propagate the new credential, which can take up to an hour to complete.
```

Comment:
I missing here: 
1. Where these credentials are located
2. How to rotate them (oc command example) 
3. Which pods need to be deleted and how. 
4. How to verify it was rotated (for example how to check main secrets in cloud-credentials, router, registry, etc) 

This does not help average ser to understand what is needed to be done.